core/lib/components/balena/sdk.js

```diff
@@ -418,4 +447,99 @@ module.exports = class BalenaSDK {
 				.get('current_services'),
 		);
 	}
+
+	/** Pushes a release to an application, from a given directory 
+	 * @param application the balena application name
+	 * @param directory the directory name containing the docker-compose for the application and the source files
```

Comment:
```suggestion
	 * @param balena application name to push the release to
	 * @param path to directory name containing the docker-compose/Dockerfile for the application and the source files
```

Reply:
@param adds the parameter name already to the docs. You just need to add the description. Looks something like this. 
![Screenshot from 2021-05-13 22-49-53](https://user-images.githubusercontent.com/22801822/118161730-8dcd9f80-b43d-11eb-8123-9ddc591d8a48.png)